packages/jest-repl/src/cli/repl.ts

```diff
@@ -5,30 +5,34 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
-import type {GlobalConfig, ProjectConfig} from 'types/Config';
-
-declare var jestGlobalConfig: GlobalConfig;
-declare var jestProjectConfig: ProjectConfig;
-declare var jest: Object;
+declare var jestGlobalConfig: Config.GlobalConfig;
+declare var jestProjectConfig: Config.ProjectConfig;
 
 import path from 'path';
 import repl from 'repl';
 import vm from 'vm';
+import { Transformer } from '@jest/transform';
+import { Config } from '@jest/types';
 
-let transformer;
+let transformer: Transformer;
 
-const evalCommand = (cmd, context, filename, callback, config) => {
+const evalCommand: repl.REPLEval = (
+  cmd: string,
+  _context: any,
+  _filename: string,
+  callback: (e: Error | null, result?: any) => void,
+  _config?: any
```

Comment:
So the type `repl.REPLEval` doesnt have the last argument `config`. Do we actually need that? We dont make use of that argument anywhere. Or any of the underscored arguments for that matter.

Reply:
Also, having the config as the last argument kind of goes against the nodejs callback model of having the callback as the last argument. Just something that I though of was abit off.